txpool/txpool.go

```diff
@@ -202,7 +202,7 @@ func NewTxPool(
 		logger:      logger.Named("txpool"),
 		forks:       forks,
 		store:       store,
-		executables: newPricedQueue(),
+		executables: (&pricedQueue{}).init(0, nil),
```

Comment:
I would leave the constructor function, and in it call this `init` function.

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1696/commits/a9c2c519b9c25b0b36613bfb3802810172b957bb
init is removed, on each `Prepare` `newPricedQueue` is created